src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1093,18 +1094,21 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
         $defaultDir = $container->getParameterBag()->resolveValue($config['default_path']);
         $rootDir = $container->getParameter('kernel.root_dir');
         foreach ($container->getParameter('kernel.bundles_metadata') as $name => $bundle) {
-            if ($container->fileExists($dir = $bundle['path'].'/Resources/translations')) {
+            if (\file_exists($dir = $bundle['path'].'/Resources/translations')) {
```

Comment:
Shouldn't we use `is_dir()` here instead of `file_exists()`?

Reply:
That would be better, yes. I'll change it!